OPConsultation-output.json

```diff
@@ -0,0 +1,2874 @@
+          "type": "Practitioner"
+        },
+        "dosageInstruction": [
+          {
+            "doseAndRate": [
+              {
+                "doseQuantity": {
+                  "value": 500,
+                  "code": "mg",
+                  "system": "http://unitsofmeasure.org",
+                  "unit": "mg"
+                },
+                "rateQuantity": {
+                  "value": 3,
+                  "code": "{1/d}",
+                  "system": "http://unitsofmeasure.org",
+                  "unit": "1/d"
+                }
+              }
+            ]
+          }
+        ]
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:a8ba176c-ce49-4517-a65f-432b724259b6",
+      "resource": {
+        "resourceType": "MedicationRequest",
+        "id": "a8ba176c-ce49-4517-a65f-432b724259b6",
+        "meta": {
+          "versionId": "0",
+          "lastUpdated": "2024-08-06T12:18:11.223+05:30",
+          "profile": [
+            "https://nrces.in/ndhm/fhir/r4/StructureDefinition/MedicationRequest"
+          ]
+        },
+        "status": "active",
+        "intent": "plan",
+        "medicationCodeableConcept": {
+          "coding": [
+            {
+              "code": "785686003",
+              "system": "http://snomed.info/sct",
+              "display": "Consolidation chemotherapy"
+            }
+          ],
+          "text": "Consolidation chemotherapy"
+        },
+        "subject": {
+          "reference": "urn:uuid:c4d052b5-2d9f-4ebf-b617-764efffa08de",
+          "type": "Patient"
+        },
+        "authoredOn": "2024-08-06T12:18:11+05:30",
+        "requester": {
+          "reference": "urn:uuid:41295111-04f9-4b83-b186-ef2975db1c7e",
+          "type": "Practitioner"
+        },
+        "dosageInstruction": [
+          {
+            "doseAndRate": [
+              {
+                "doseQuantity": {
+                  "value": 500,
+                  "code": "mg",
+                  "system": "http://unitsofmeasure.org",
+                  "unit": "mg"
+                },
+                "rateQuantity": {
+                  "value": 3,
+                  "code": "{1/d}",
+                  "system": "http://unitsofmeasure.org",
+                  "unit": "1/d"
+                }
+              }
+            ]
+          }
+        ]
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:dc0d79a6-2611-4917-9d34-e6a98acede05",
+      "resource": {
+        "resourceType": "MedicationRequest",
+        "id": "dc0d79a6-2611-4917-9d34-e6a98acede05",
+        "meta": {
+          "versionId": "0",
+          "lastUpdated": "2024-08-06T12:18:11.223+05:30",
+          "profile": [
+            "https://nrces.in/ndhm/fhir/r4/StructureDefinition/MedicationRequest"
+          ]
+        },
+        "status": "active",
+        "intent": "plan",
+        "medicationCodeableConcept": {
+          "coding": [
+            {
+              "code": "367336001",
```

Comment:
367336001 - is code for chemotherapy procedure, not a drug. If you want to put a chemo drug, please find out what chemo drug is usually given and find appropriate code. Please do not put random data! 

for Acute Myeloid leukemia - Cytarabine (substance) is a better example